src/common/transformations/src/transformations/common_optimizations/transpose_sinking_utils.cpp

```diff
@@ -129,17 +129,33 @@ ov::Output<ov::Node> FixInputNodeRank(ov::Output<ov::Node> input_node, ov::Rank:
 }  // namespace
 
 namespace sink_forward {
+AxisVector AlignTransposeOrder(const Output<Node>& output, const TransposeInputsInfo& transpose_input_info) {
+    auto num_of_val = static_cast<int64_t>(shape_size(transpose_input_info.transpose_const->get_shape()));
```

Comment:
should we check that `transpose_input_info.transpose_const` is not `nullptr`?

Reply:
fixed

Reply:
but actually it was invoked only from `UpdateInputTransposes` function where we have a check:
![image](https://user-images.githubusercontent.com/21181928/216941974-9b17b407-114e-49d8-8611-d60043f23c36.png)